### ISSUE TITLE
Prism.js is loaded over http so we got a "mixed content" error...

### ISSUE THREAD

**Issue by shulard** [hoaproject/Blog#11]
Hello,

I've checked the blog online and found an error regarding prism.js library.

For example on that URL : https://blog.hoa-project.net/2017/12-Year-in-review.html

I got an error in my console :

```
Blocage du chargement du contenu mixte actif (mixed active content) « http://prismjs.com/prism.js »
```

Since the blog is served over HTTPS, this is not a strange error. Maybe the better way to handle this is to install locally Prism.

It has already been done on the hoaproject/W3 repository, is there a way to reference that file or do we need to install a specific blog version ?

**Comment by Hywan** [comment]
Indeed. In https://github.com/hoaproject/Blog/blob/316b82854c8d8ecb8e27f2d09ad822c6a8c3fac7/Source/Overlays/Article.xyl#L26, we should link to https://static.hoa-project.net/Javascript/Prism.js.

Can you do that?

**Comment by shulard** [comment]
I've created the PR 😄.

**Comment by shulard** [comment]
This was fixed with the #13 PR.
